<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pushButton_Pic_Text</class>
 <widget class="QMainWindow" name="pushButton_Pic_Text">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pushButton_Pic_Text</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>4</y>
      <width>284</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>68</y>
      <width>284</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string>ToolButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>140</y>
      <width>284</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
